### DISCUSSION TITLE
Cloudflare Domain Invalid SSL certificate Error with code 526

### DISCUSSION THREAD

**Discussion by fiqryq** [Dokploy/dokploy#1035] [Q&A] [answered]
Hi guys , I’m new to using self-hosted services like Dokploy. This is my first time trying to deploy Umami using the Dokploy template app. When I attempted to add a custom domain, I encountered an issue. Here's the error I got when trying to add the custom domain :

![CleanShot 2024-12-31 at 00 57 16](https://github.com/user-attachments/assets/0ae027e4-42de-4b51-ab91-a9e989f40f94)

**Here my config :**

![CleanShot 2024-12-31 at 00 55 46](https://github.com/user-attachments/assets/002de89a-94b8-42e3-83d0-4bf087db1dbd)
![CleanShot 2024-12-31 at 01 01 24](https://github.com/user-attachments/assets/147e15a1-8f96-4956-91cc-826b9ab0166e)

**dokploy settings :**
![CleanShot 2024-12-31 at 00 56 34](https://github.com/user-attachments/assets/126e5a26-bdf2-4688-985a-2fc4d4b205e1)

![CleanShot 2024-12-31 at 01 07 29](https://github.com/user-attachments/assets/b095860a-cfbb-4287-a191-ef88d9cc886b)

**Note :**
Initially, I left the Dokploy settings blank, as shown in the image below. However, I encountered a "404 Page Not Found" error.

![CleanShot 2024-12-31 at 01 03 32](https://github.com/user-attachments/assets/de0a2d07-dae6-45cc-bf5b-86561555ea20)

![CleanShot 2024-12-31 at 01 04 05](https://github.com/user-attachments/assets/0f1390a1-c296-4dbf-a4e0-f5b135e9cfd7)

also i try settings like this with disable ssl and its keep error :
![CleanShot 2024-12-31 at 01 08 25](https://github.com/user-attachments/assets/9103ec8f-1724-4f56-908e-c5c0e60ac5ab)

**Comment by ravinataraju** [comment]
try re-deploying the application after domain name change was made and wait for the container to come up and check in logs. secondly i had to configure and save the strict full on cloudflare twice yesterday and my services started to work. cloudflare propagation sometime took time and showed that screen, but it resolved after a few minutes

**Reply by fiqryq** [reply]
its still not working

**Comment by fiqryq** [comment, marked as answer]
its working , i just need to redeploy : 
![CleanShot 2024-12-31 at 20 10 32](https://github.com/user-attachments/assets/0e3a545a-7fbb-419a-ab88-83902f147ac1)
